dice.go

```diff
@@ -34,12 +33,20 @@ func rollDice(code string) (*diceRolls, error) {
 		if err != nil {
 			return nil, fmt.Errorf("Could not parse a number of dice from '%s'", numberStr)
 		}
+		if number > 20 {

```

Comment:
Thanks for the feedback&PR!
If a user asks for more than 20 rolls with a sum request, they might not notice the cap and just use the wrong sum, so I'd rather have the command fail with an explicit error message.

Reply:
Sounds good, I'll update the patch.